src/utils/hooks/useCreateReview.tsx

```diff
@@ -24,8 +30,10 @@ const useCreateReview: UseCreateReview = ({ reviewStructure, reviewAssignment })
   const constructReviewInput: ConstructReviewInput = (structure) => {
     const elements = Object.values(structure?.elementsById || {})
     // Only create review for elements that are pendingReview and are assigned
+    // and are reviewable
     const reviewableElements = elements.filter(
-      ({ isPendingReview, isAssigned }) => isPendingReview && isAssigned
+      ({ isPendingReview, isAssigned, element: { isReviewable } }) =>
+        isPendingReview && isAssigned && isReviewable !== IsReviewableStatus.Never
```

Comment:
The only bit of logic required -- don't create review responses for elements that are NEVER reviewable.